plottable.js

```diff
@@ -3,7 +3,7 @@ Plottable 1.14.0 (https://github.com/palantir/plottable)
 Copyright 2014-2015 Palantir Technologies
 Licensed under MIT (https://github.com/palantir/plottable/blob/master/LICENSE)
 */
-
+//# sourceURL=plottable.js
```

Comment:
Sorry my mistake leaving that in there - I've been working with your (elegant and ingenious) quicktests... this line enables the chrome developer tools to display plottable.js in the sources page, even when it is dynamically loaded. this makes it easier to set breakpoints. I think it would be a useful thing to have in there for that reason (and similarly, in each of the quicktests) but - that's a separate issue for another day.